.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm-3.12
+    - 5.6
+    - 7.0
+    - 7.1
+    - nightly
+    - hhvm-3.12
 
 matrix:
-  allow_failures:
```

Comment:
Stick to 2 spaces. There's no reason to change things based on a single persons opinion here as it just pollutes history. Additionally the auto for after uses 2 spaces for yaml afaik.